docs/develop/financial-accounting.md

```diff
@@ -0,0 +1,187 @@
+types of these accounts look different depending on whether you are considering them from the
+perspective of you or the bank.
+
+Now, back to our original questions: is the loan balance $100 or -$100 and is the bank account
+balance $200 or -$200? On some level, this feels a bit arbitrary.
+
+Wouldn't it be nice if there were some **commonly agreed-upon standards** so we wouldn't have to
+make such an arbitrary decision? Yes! This is exactly what debits and credits and the financial
+accounting type system provide.
+
+## Types of Accounts
+
+In financial accounting, there are 5 main types of accounts:
+
+- **Assets** - what you own
+- **Liabilities** - what you owe to other people
+- **Equity** - amount invested by the owners or shareholders
+- **Revenue** - money coming in
+- **Expenses** - money going out
+
+As mentioned above, the type of account depends on whose perspective you are doing the accounting
+from. In those examples, the loan you have from the bank is liability for you, because you owe the
+amount to the bank. However, that same loan is an asset from the bank's perspective. In contrast,
+the money in your bank account is an asset for you but it is a liability for the bank.
+
+Each of these major categories are further subdivided into more specific types of accounts. For
+example, in your personal accounting you would separately track the cash in your physical wallet
+from the funds in your checking account, even though both of those are assets. The bank would split
+out mortgages from car loans, even though both of those are also assets for the bank.
+
+## Double-Entry Bookkeeping
+
+Categorizing accounts into different types is useful for organizational purposes, but it also
+provides a key error-correcting mechanism.
+
+Every record in our accounting is not only recorded in one place, but in two. This is double-entry
+bookkeeping. Why would we do that?
+
+Let's think about the bank loan in our example above. When you took out the loan, two things
+actually happened at the same time. On the one hand, you now owe the bank $100. At the same time,
+the bank gave you $100. These are the two entries that comprise the loan transaction.
+
+From your perspective, your liability to the bank increased by $100 while your assets also increased
+by $100. From the bank's perspective, their assets (the loan to you) increased by $100 while their
+liabilities (the money in your bank account) also increased by $100.
+
+Double-entry bookkeeping ensures that funds are always accounted for. Money never just appears.
+**Funds always go from somewhere to somewhere.**
+
+## Keeping Accounts in Balance
+
+Now we understand that there are different types of accounts and every transaction will be recorded
+in two (or more) accounts -- but which accounts?
+
+The [Fundamental Accounting Equation](https://en.wikipedia.org/wiki/Accounting_equation) stipulates
+that:
+
+**Assets = Liabilities + Equity**
```

Comment:
We could say "Assets - Liabilities = Equity" reason being that this is what the owners own (or owe!) w.r.t. the business after all is said and done.